recipe/meta.yaml

```diff
@@ -43,14 +44,13 @@ requirements:
     - llvm-spirv-{{ llvm_version.split(".")[0] }}
 
     - zlib                               # [osx]
-    - ld64 >=530                         # [osx]
+    - ld64 >=530,<711                     # [osx]
   run:
     - llvm-spirv-{{ llvm_version.split(".")[0] }}
     - ocl-icd                            # [linux]
     - khronos-opencl-icd-loader          # [osx]
-    - ld64 >=530                         # [osx]
+    - ld64 >=530,<711                     # [osx]
```

Comment:
Add a link to an issue in ld64 or its feedstock justifying the upper bound. (so that there's a way to know when the upper bound can be removed) Same above.

Reply:
added in [edf3611](https://github.com/conda-forge/pocl-feedstock/pull/96/commits/edf36115f23e928de83c600faf6bf2776ec8cc8b)